pkg/test/store/postgres/test_container.go

```diff
@@ -0,0 +1,110 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/kumahq/kuma/pkg/config"
+	pg_config "github.com/kumahq/kuma/pkg/config/plugins/resources/postgres"
+	"github.com/kumahq/kuma/pkg/core"
+	"github.com/kumahq/kuma/pkg/plugins/common/postgres"
+)
+
+type PostgresContainer struct {
+	container testcontainers.Container
+	WithSsl   bool
+}
+
+func resourceDir() string {
+	dir, _ := os.Getwd()
+	for path.Base(dir) != "pkg" && path.Base(dir) != "app" {
+		dir = path.Dir(dir)
+	}
```

Comment:
err, what?

Reply:
It's finding the root directory of the project. Do you have another way of doing this?